lib/components/helpers/pretty-text-input.js

```diff
@@ -357,7 +363,9 @@ export default createReactClass({
 
     var textBox = this.textBoxRef;
     textBox.innerHTML = ''; // release any previous read-only content so it can be GC'ed
-
+    this.setState({
```

Comment:
@efhjones Did you try setting this synchronously and have problems? In other words, just setting a local variable and then setting the position after creating codemirror? It looks like there's some possible async business in `tagCodeMirror`, but you could just pass the position along to that function and set the position after `this.codeMirror.operation(tagOps)`.

Reply:
No I didn't have any problems @jdeal but was worried about that so I was hoping to get/set it only when needed. However your suggestion above makes sense! I'll rework it, thanks!